hitbox-backend/src/value.rs

```diff
@@ -0,0 +1,51 @@
+use chrono::{DateTime, Utc};
+
+#[cfg_attr(test, derive(Clone, PartialEq, Debug))]
+pub struct CachedValue<T> {
```

Comment:
What do you think - maybe we should rename CachedValue to ExpiredValue? It's easy to get confused with the CacheableResponse, CachedValue etc.

Reply:
Okay, you're right. 
But CachedValue is just a value with an extra `expired` field.
Because of that, I think we should rename it.